app/js/components/common/navigate.jsx

```diff
@@ -10,10 +10,12 @@
 
 import React, { PropTypes } from 'react';
 
+import './navigateStyle';
+
 export  const Navigate = ({currentPage, totalPage, navigatePage}) => {
   return(
     <div className="tableNavigation">
-      <button className="btn btn-primary" onClick={navigatePage} value="first">FIRST</button>
+      <button className="btn btn-primary" onClick={navigatePage} id="first" value="first">FIRST</button>
```

Comment:
Since the button properties spread to another line, can you break the properties/attributes to be on separate lines?

Reply:
Please have a look at it again @andela-dbamidele , it does not span to the next line, it is within good length to remain as a single line.

Reply:
Oh cool. Maybe it's the way I'm viewing it on github. 

Great job. 👍